scripts/tests/cirque_tests.sh

```diff
@@ -35,7 +44,7 @@ function __socat_clean() {
     socat_pid=$(ps aux | grep "[s]ocat" | awk '{print $2}')
     if [ ! -z "$socat_pid" ]; then
         for pid in "$socat_pid"; do
-            kill -2 "$pid"
+            sudo kill -2 "$pid"
```

Comment:
why sudo?

Reply:
Socat is executed by cirque, which is spawned under root.

Reply:
I wonder if running this script under sudo is acceptable

Reply:
cirque's virtual wifi and thread needs the root permission, soon virtual ble also need the sudo permission. These virtual wifi, thread, ble are relying on linux modules, mac80211_hwsim, otbrposix, btvirt along with bluetoothd